rero_ils/theme/templates/rero_ils/macros/macro.html

```diff
@@ -173,10 +173,14 @@
     {{field(class_="form-control", placeholder=placeholder, **extras)}}
     {%- if icon %}
     <div class="input-group-append">
-    <span class="input-group-text">
-      <i class="{{icon}} form-control-feedback" aria-hidden="true" ></i>
-    </span>
-  </div>
+      <span class="input-group-text">
+        {% if field.id != 'password' %}
+        <i class="{{icon}} form-control-feedback" aria-hidden="true"></i>
+        {% else %}
+        <small><i id="password-show-hide" class="fa fa-eye form-control-feedback" aria-hidden="true" title="{{ _('Show password') }}"></i></small>
```

Comment:
1. `<small>` makes the icon difficult to click. Ideally, make the whole gray square clickable to show the password, not only the icon.
2. Can we make the pointer like a link when hovering over the clickable area ? Otherwise, it is not clear that I can click.

Great work though! 👏🏻